src/lib.rs

```diff
@@ -148,6 +157,39 @@ impl Game {
             if inputs.right {
                 player.x += self.speed * time_steps_passed
             }
+
+            if !inputs.q && self.last_input.q {
```

Comment:
Am I reading this right and this kills on keyup?

Killing on keydown feels more responsive I think